lib/src/lifecycle_module.dart

```diff
@@ -159,6 +163,9 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
   @protected
   Span get activeSpan => _activeSpan;
 
+  /// If this module has had an overridden name and therefor is being traced
+  bool get isTraced => name != _defaultName;
```

Comment:
There was no way to know that a module is actually being traced externally, unless we tried to guess at this default name. We figured a private string is liable to change, though, so it's better to expose this property to consumers.

Reply:
What's the use-case for telling whether a module is traced?

Also, this will only ever be `true` in minified dart2js, and not other runtimes.

Reply:
Name won't equal default name if we override the name property with a hard coded string, right? My understanding was that overriding the default name with a hard coded string would allow the module to be traced with a meaningful span prefix.

Here is our use case, where a parent listens to both module isFirstUseful and manual spans to determine that it is itself firstUseful:
https://github.com/Workiva/graph_app/pull/7145/files#diff-67c7c5d0ea4739dbee0a0091e12a0dbdR83

Reply:
Just curious, what's the use case for this?

Reply:
It's for tracking child module spans to determine the parent is first useful. If the module didn't have the `name` property overriden, we'd have no way to know that it's impossible for that module to trigger the `isFirstUseful` so our parent's `isFirstUseful` would also never trigger.

Reply:
Wait, huh? A module name should only ever be informative for logging purposes, we shouldn't be relying on that for behavior in code. What if it changes?

Reply:
This is the origination: https://github.com/Workiva/w_module/blob/master/lib/src/lifecycle_module.dart#L175

It's a necessary evil since we obfuscate the JS. Without it, we'd have no meaningful naming for the lifecycle event spans.

Reply:
To be clear: We don't care what the name is necessarily, we just care if it is set because that means that it's being traced and we're able to feed it's "firstUseful" into the parents "firstUseful"

Reply:
Ah, my bad, I was mistaken; the default name of is `LifecycleModule(${this.runtimeType})`, so you're correct in that any named module will be traced.

@georgelesica-wf we disable tracing when a custom name has not been set, because otherwise the minified name would be used in dart2js and you'd get traces with names like `LifecycleModule(jfkdlas)`.

Reply:
Oh OK, gotcha, I wasn't aware there was anything related to tracing built-in here.